unison-src/transcripts/fix2254.output.md

```diff
@@ -64,24 +67,24 @@ unique type A a b c d
 Let's do the update now, and verify that the definitions all look good and there's nothing `todo`:
 
 ```ucm
-.a2> update.old
+scratch/a2> update.old
 
   ⍟ I've updated these names to your new definition:
   
     type A a b c d
 
-.a2> view A NeedsA f f2 f3 g
+scratch/a2> view A NeedsA f f2 f3 g
 
   type A a b c d
-    = B b
+    = A a
```

Comment:
I'm not positive what determines the uuids used for determining constructor order tbh 🤷🏼‍♂️